package.json

```diff
@@ -6,8 +6,11 @@
   "scripts": {
     "dev": "nodemon --exec babel-node src/index.js",
     "start": "node dist/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "babel src --out-dir dist"
+    "build": "babel src --out-dir dist",
+    "sync-env": "sync-dotenv",
+    "test": "nyc --reporter=text mocha ./tests/ --recursive -r @babel/register",
+    "coveralls": "nyc report --reporter=lcov --reporter=text-lcov | ./node_modules/coveralls/bin/coveralls.js ",
```

Comment:
Why do we have `./node_modules/coveralls/bin/coveralls.js` and not `coveralls`. Also, please add `husky` to the app so that we can sync our .env file before every commit.

Install husky as a dev dependency
```
npm install husky -D
```

In our package.json
```
"devDependencies": {
...
},
"husky": {
    "hooks": {
      "pre-commit": "npm run sync-env",
    }
}
```

Reply:
Ok. I have added the husky and the prehook